rust/hyperlane-core/src/config/config_path.rs

```diff
@@ -76,10 +76,10 @@ impl ConfigPath {
 
     /// Get the environment variable formatted path.
     pub fn env_name(&self) -> String {
-        ["HYP", "BASE"]
+        ["HYP"]
             .into_iter()
             .chain(self.0.iter().map(|s| s.as_str()))
-            .map(|s| s.to_uppercase())
```

Comment:
It's unclear to me why some parsing logic uses flat case, while in here we use `Case::UpperFlat`

Reply:
It is purely styalistic to display env vars as `HYP_KEY_TO_THING` instead of `HYP_key_to_thing`.